src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -257,6 +258,10 @@ def get_clouds():
         if c.profile is None:
             # If profile isn't set, use latest
             setattr(c, 'profile', 'latest')
+        if c.profile not in API_PROFILES:
+            # The profile doesn't exist. Maybe it was renamed or is user error.
+            logger.info('Profile %s does not exist, using latest.', c.profile)
```

Comment:
If I make this a warning, it will appear multiple times as this method is called several times.
Didn't want to create a global variable to keep track of if the log message has been printed already.
Thoughts?